### PR TITLE
use npm-run-all instead of concurrently

### PR DESCRIPTION
**Pull request by erikyuzwa**
- Tackles https://github.com/slashman/jsrl/issues/13
- Make use of `npm-run-all` which is feeling more solid than `concurrently`
- Needed to add a small delay (`5s`) ONLY on DEV startup with Electron
- `npm run electron` and `npm run electron:pixi` are still the 2 ways to run a dev Electron server

### CODE REVIEW

**Comment by slashman** [comment]
Obviously not very happy about the delay :/ but if this improves the situation at least, I think we should put it in.

File: package.json [hunk 1]
```diff
@@ -8,8 +8,12 @@
     "build:electron:pixi": "cross-env NODE_ENV=production --graphics=pixi --target=electron-renderer webpack --mode=production",
     "build:web": "webpack --target=web --mode=production",
     "build:web:pixi": "cross-env --graphics=pixi webpack --mode=production",
-    "electron": "concurrently \"cross-env NODE_ENV=development --target=electron-renderer webpack --watch\" \"cross-env NODE_ENV=development electron .\"",
-    "electron:pixi": "concurrently \"cross-env NODE_ENV=development --graphics=pixi --target=electron-renderer webpack --watch\" \"cross-env NODE_ENV=development electron .\"",
+    "delay": "delay 5",
+    "electron:dev": "cross-env NODE_ENV=development --target=electron-renderer webpack --watch",
```
Comment (slashman) [inline package.json:12]: I wonder if there's a way to mark scripts that are not meant to be run directly by the game developer, but only by other scripts. I guess we are getting into the territory of grunt/gulp here.

Comment (erikyuzwa) [reply package.json:12]: that's an interesting comment - I really agree. Something to think about, but I have no suggestion at the moment.

**Comment by erikyuzwa** [comment]
> Obviously not very happy about the delay :/ but if this improves the situation at least, I think we should put it in.

Yeah I hear you - it IS only on the first startup...but it feels 🧀

**Comment by erikyuzwa** [comment]
any interest in this @slashman or should I just shut it down?

**Comment by erikyuzwa** [comment]
I'm thinking `electron-forge` just might fix this issue with the way it works. Killing this one @slashman

**Pull request closed without merging**